src/integrationTest/java/uk/gov/hmcts/reform/orgrolemapping/controller/RefreshControllerRefreshJobIntegrationTest.java

```diff
@@ -285,29 +281,27 @@ public void shouldProcessRefreshRoleAssignmentsWithJobIdToPartialComplete_status
                 isRefreshJobInStatus(jobId, ABORTED)));
 
         logger.info(" -- Refresh Role Assignment record updated successfully -- ");
-        RefreshJobEntity refreshJob = getRecordsFromRefreshJobTable(jobId);
-        assertEquals(ABORTED, refreshJob.getStatus());
-        assertNotNull(refreshJob.getUserIds());
-        assertThat(refreshJob.getLog(), containsString(String.join(",", refreshJob.getUserIds())));
+        RefreshJobEntity afterRefreshJob = getRecordsFromRefreshJobTable(jobId);
+        assertEquals(ABORTED, afterRefreshJob.getStatus());
+        assertNotNull(afterRefreshJob.getUserIds());
+        assertThat(afterRefreshJob.getLog(), containsString(String.join(",", afterRefreshJob.getUserIds())));
     }
 
-    //@Disabled("Intermittent DTSAM-111")
     @Test
     @Order(5)
-    @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts = {"classpath:sql/insert_refresh_jobs.sql"})
     public void shouldProcessRefreshRoleAssignmentsWithFailedUsersToComplete() throws Exception {
         when(securityUtils.getServiceName()).thenReturn(AUTHORISED_JOB_SERVICE);
 
         logger.info(" RefreshJob record With JobId and failed UserIds to process successful");
-        Long jobId = 3L;
-
-        //Verify before job
-        RefreshJobEntity refreshJob = getRecordsFromRefreshJobTable(jobId);
-        assertEquals("NEW", refreshJob.getStatus());
-        RefreshJobEntity linkedJob = getRecordsFromRefreshJobTable(refreshJob.getLinkedJobId());
-        assertNotNull(linkedJob.getUserIds());
-        assertEquals(ABORTED, linkedJob.getStatus());
-        assertNotEquals(0, linkedJob.getUserIds().length);
+        //Long jobId = 3L;
```

Comment:
```suggestion
```

should this be removed? as `jobId` is retrieved from entity on line 304

Reply:
well spotted, removed